src/index.ts

```diff
@@ -1,40 +1,50 @@
 import { types, ExecuteWithScope, createExecuteWithScope } from "brs";
-import { glob } from "glob";
+import fg from "fast-glob";
 import * as path from "path";
-import * as util from "util";
 import * as c from "ansi-colors";
 import { ReportOptions } from "istanbul-reports";
 import { reportCoverage } from "./coverage";
-import { formatInterpreterError } from "./util";
+import { formatInterpreterError, globMatchFiles } from "./util";
 import { createTestRunner, ReporterType } from "./runner";
 
 const { isBrsBoolean, isBrsString, RoArray, RoAssociativeArray } = types;
-const globPromise = util.promisify(glob);
 
-interface Options {
+interface CliOptions {
+    /** The test reporter to use. */
     reporter: ReporterType;
+    /** A path to a file that we should load into global exec scope prior to test run. */
     requireFilePath: string | undefined;
-    forbidFocused: boolean;
+    /** Whether or not to fail the test run if focused cases are detected. */
+    forbidFocused?: boolean;
+    /** The istanbul coverage reporters to use. */
     coverageReporters?: (keyof ReportOptions)[];
+    /** The directory where we should load source files from, if not 'source'. */
+    sourceDir?: string;
+    /**
+     * A list of strings to match files against, specified in the command.
+     * If empty, we will test/search for all *.test.brs files.
+     */
+    filePatterns: string[];
 }
 
-async function findBrsFiles(sourceDir: string | undefined) {
+async function findBrsFiles(sourceDir?: string) {
     let searchDir = sourceDir || "source";
     const pattern = path.join(process.cwd(), searchDir, "**", "*.brs");
-    return globPromise(pattern);
+    return fg.sync(pattern);
```

Comment:
If we're going to return this glob synchronously, we may as well make `findBrsFiles` not an `async` function.  Or keep it `async` and return a `Promise`!  Doing both feels a bit weird :)

Reply:
Great call!